articles/libraries/auth0js/v9/migration-guide.md

```diff
@@ -0,0 +1,49 @@
+---
+section: libraries
+title: Migrating to Auth0.js v9
+description: How to migrate to auth0.js v9
+toc: true
+---
+# Migrating to Auth0.js v9
+
+Auth0.js v9 has been improved to work better in embedded login scenarios. It operates with enhanced security and removes dependencies that have been deprecated as per Auth0's roadmap. In some cases, these security enhancements may impact application behavior when upgrading from an earlier versions of Auth0.js. 
+
+Auth0.js can be used to implement authentication in different ways:
+
+- Centralized Login, using the `.authorize()` method, where the user is redirected from your website to an Auth0's Hosted Login Page.
+
+- Embedded Login, using the `.login()`, `popup.loginWithCredentials()`, `client.loginWithCredentials()` methods, where the login dialog is displayed in the application's website.
```

Comment:
remove `client.loginWithCredentials()`. We have it just to be backwards compatible. new apps should use just `.login`

Reply:
The idea is to communicate "if I'm using some of those methods then I need to worry about migrating", so I think we need to keep it